src/Corax/Queries/TermMatch.cs

```diff
@@ -10,6 +10,8 @@
 using System.Collections.Generic;
 using Sparrow.Compression;
 using Sparrow.Server;
+using System.Security.Cryptography;
```

Comment:
Is this necessary?

Reply:
No, I dont even know how that end up there.